bundles/net.tourbook.device/src/net/tourbook/device/sporttracks/FitLogSAXHandler.java

```diff
@@ -536,7 +543,8 @@ private void finalizeTour() {
          tourData.setWeather_Precipitation(_currentActivity._weatherPrecipitation);
       }
 
-      if (!_currentActivity._customDataFields.isEmpty()) {
+
+      if (_currentActivity._customDataFields.size() > 0) {
 
          final StringBuilder tourNotes = new StringBuilder(tourData.getTourDescription());
 
```

Comment:
This string occurs a few times 

`StatusUtil.log(... + ": FitlogEx parse error", ...);` 

it could put in a constant 

`"%s: FitlogEx parse error"`

Reply:
See the [S1192](https://rules.sonarsource.com/java/RSPEC-1192) rule

Reply:
ok I'll create a CONSTANT for "FitlogEx parse error"